client/scripts/views/components/component_kit/cw_engagement_button.tsx

```diff
@@ -37,10 +37,9 @@ export const CWEngagementButton: m.Component<EngagementButtonAttrs> = {
         })}
         onclick={onclick}
       >
-        {m(CreateIcon, {
-          size: IconSize.MD,
+        {m(CWIcon, {
```

Comment:
Can't you just reference this like a React component?
ie 
```
<button ...>
  <CWIcon ... />
  <span>{label}</span>
</button>
```

Reply:
That change happens in https://github.com/hicommonwealth/commonwealth/pull/1669.